Code/Libraries/RESTLibrary/build.gradle

```diff
@@ -13,12 +13,10 @@ apply plugin: 'idea'
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 
-
 jar { enabled = true }
```

Comment:
I don't think these are necessary

Reply:
we need restLibrary to be compiled as jar so that other gradle projects can import it no?